### PR TITLE
Integrate shipping calculator with graphql resolver

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?
Integrate the `ShippingSimulator` component with the newest release of `store-graphql` and use real data from the backend.

#### What problem is this solving?
The component was using only mocked data.

#### How should this be manually tested?
Access my [workspace](https://lucas--storecomponents.myvtex.com/porsche-911/p), the id of the product (used for the query) is mocked in the workspace, but not in the pull request.

#### Screenshots or example usage
screen shot 2018-05-29 at 16 14 23

```jsx
<ShippingCalculator
  skuId="3" // These props are part of the new component api
  seller="1"
/>
```

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ShippingSimulator/index.js [hunk 1]
```diff
@@ -41,28 +51,35 @@ class ShippingSimulator extends Component {
...
-          name: 'Retirada Rápida',
-        },
-      ],
+      loading: true,
+    })
+
+    this.props.client.query({
+      query: getShippingEstimates,
+      variables: {
+        country: 'BRA', // TODO @lucasecdb: get country of user instead of hard-coding
```
Comment (andreldsa) [inline react/components/ShippingSimulator/index.js]: Think this must be from the Pages-Editor

File: react/components/ShippingSimulator/components/ShippingTable.js [hunk 2]
```diff
@@ -1,26 +1,52 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
+import { FormattedMessage } from 'react-intl'
 
 import ShippingTableRow from './ShippingTableRow'
 
 export default class ShippingTable extends Component {
   static propTypes = {
     /** Placeholder */
```
Comment (salesfelipe) [inline react/components/ShippingSimulator/components/ShippingTable.js]: Placeholder?

File: react/components/ShippingSimulator/components/ShippingTableRow.js [hunk 3]
```diff
@@ -3,29 +3,29 @@ import PropTypes from 'prop-types'
...
 
-  if (value === undefined) {
+  if (price === undefined) {
     valueText = '-'
-  } else if (value === 0) {
+  } else if (price === 0) {
     valueText = intl.formatMessage({ id: 'shipping.free' })
   } else {
-    valueText = intl.formatNumber(value, { style: 'currency', currency: 'BRL' })
+    valueText = intl.formatNumber(price, { style: 'currency', currency: 'BRL' })
```
Comment (salesfelipe) [inline react/components/ShippingSimulator/components/ShippingTableRow.js]: Is this always BRL, or should be according to user's location?

File: react/components/ShippingSimulator/components/ShippingTable.js (hunk 2 above)
Comment (lucasecdb) [reply react/components/ShippingSimulator/components/ShippingTable.js]: good catch

File: react/components/ShippingSimulator/components/ShippingTableRow.js (hunk 3 above)
Comment (lucasecdb) [reply react/components/ShippingSimulator/components/ShippingTableRow.js]: I believe so

**Pull request merged**